two_factor/templates/two_factor/twilio/sms_message.html

```diff
@@ -0,0 +1,3 @@
+{% load i18n %}
+{% trans sms_message %}
+{% trans 'Your token is '%}{{ token }}
```

Comment:
This should use `blocktrans` so translators have some context. Also the file should end with a newline.

Reply:
done